keyboards/k_type/config.h

```diff
@@ -15,30 +15,56 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef CONFIG_H
-#define CONFIG_H
+#pragma once
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0x1c11
-#define PRODUCT_ID      0xb04d
-#define DEVICE_VER      0x0001
+#define VENDOR_ID    0x1C11
+#define PRODUCT_ID   0xB04D
+#define DEVICE_VER   0x0001
 #define MANUFACTURER Input Club
-#define PRODUCT K-Type/QMK
+#define PRODUCT      K-Type
```

Comment:
I seem to remember that the `/QMK` suffix was something we decided with the Input:Club team to make it easier to debug which firmware was running for them.